package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -433,6 +456,30 @@ pvc.BaseChart
             }
         }
 
+        // CDF603
+        var width;
+        if(axis.option.isDefined('FixedLength')) {
+            width = axis.option('FixedLength');
+            if(width < 0) width = -width; // Length is always an absolute value
```

Comment:
This kind of normalization should be done by the option's own cast function.
Also, getting the absolute value doesn't seem right - it's simply invalid.
You can use `cast: def.number.toNonNegative`.